<launch>
  <arg name="port"                        default="8765" />
  <arg name="num_threads"                 default="0" />
  <arg name="topic_discovery_period"      default="5.0" />

  <arg name="topics_glob" default="[*]" />

  <node name="foxglove_server" pkg="foxglove_websocket_ros" type="server">
    <param name="port"                    value="$(arg port)"/>
    <param name="num_threads"             value="$(arg num_threads)"/>
    <param name="topic_discovery_period"  value="$(arg topic_discovery_period)"/>
  </node>

  <node name="rosapi" pkg="rosapi" type="rosapi_node">
    <param name="topics_glob" value="$(arg topics_glob)"/>
  </node>
</launch>
